features/step-definitions/search.steps.js

```diff
@@ -5,7 +5,8 @@ const homeElements = page.home.elements;
 const invalidStockSymbol = 'ASDF';
 
 module.exports = function() {
-  this.Given(/^my cursor is in the searchbar$/, () => {
+  // Scenario: Search for an invalid stock
```

Comment:
Keep in mind that this steps can be reused by multiple scenarios.  Therefore I'm not sure if they should be commented to belong to certain ones.  At the same time I like the attempt at organization.